docs/UpdatedData.md

```diff
@@ -300,4 +300,18 @@ Leveraging the `deviceEvent` type and creating 2 new subTypes with the same stru
   "inputTime": "2020-05-12T08:40:00.000Z",
   "timezoneOffset": 60
 }
+{
+  "type": "deviceEvent",
+  "subType": "warmup",
+  "eventId": "Warm123456789",
+  "duration": { 
+    "value": 3,
+    "units": "hours"
+  },
+  "deviceId": "Id12345",
+  "deviceTime": "2018-02-01T00:00:00",
+  "time": "2020-05-12T08:50:08.000Z",
+  "inputTime": "2020-05-12T08:40:00.000Z",
```

Comment:
I think we won't have an InputTime for this event

Reply:
But for now inputTime is required

Reply:
ok